shared/chat/blocking/invitation-to-block.tsx

```diff
@@ -11,23 +11,20 @@ const BlockButtons = (props: Props) => {
   const dispatch = Container.useDispatch()
   const nav = Container.useSafeNavigation()
 
-  const conversationMeta = Container.useSelector(state => state.chat2.metaMap.get(props.conversationID))
```

Comment:
a common problem of the older code. selecting on a very high level thing which changes more often than you care about